src/test/java/com/aws/greengrass/builtin/services/mqttproxy/MqttProxyIPCAgentTest.java

```diff
@@ -60,20 +57,9 @@
 class MqttProxyIPCAgentTest {
     private static final String TEST_SERVICE = "TestService";
     private static final String TEST_TOPIC = "TestTopic";
+    private static final String TEST_TOPIC_WILDCARD = "TestTopic/#";
     private static final byte[] TEST_PAYLOAD = "TestPayload".getBytes(StandardCharsets.UTF_8);
 
-    private static final String TEST_SINGLE_LEVEL_WILDCARD = "topic/with/single/level/+/wildcard";
```

Comment:
before you go deleting, make sure the old tests do pass with the new authZ code, since we know the tests worked correctly before your change.

Reply:
keep these tests or their equivalents.

Reply:
Is it okay if I have tested these cases in AuthZ as now, MqttProxy IPC just uses `isAuthorized` with `ResourceLookupPolicy.MQTT_STYLE`

Reply:
Yes, if all the same cases are covered then that is fine.